DocumentDataAPI/DocumentDataAPI/Data/Repositories/NpgDocumentContentRepository.cs

```diff
@@ -50,22 +50,22 @@ public int Add(DocumentContentModel entity)
                         });
     }
 
-    public int Delete(DocumentContentModel entity)
+    public async Task<int> Delete(DocumentContentModel? entity)
```

Comment:
Den skal da ikke være nullable her
```suggestion
    public async Task<int> Delete(DocumentContentModel entity)
```

Reply:
ved ikke hvordan den blev det